lib/liftoff/project_builder.rb

```diff
@@ -14,11 +14,43 @@ def create_project
       end
 
       xcode_project.save
-      xcode_project.generate_scheme
+      create_schemes
     end
 
     private
 
+    def create_schemes
+      xcode_project.generate_scheme
+
+      if @config.schemes
+        @config.schemes.each do |scheme_config|
```

Comment:
What are your thoughts on making a method called `schemes` that returns this config or an empty array to remove this conditional?